website/templates/project/contributors.mako

```diff
@@ -344,7 +345,9 @@
                     <!-- ko ifnot: deleteStaged -->
                         <!-- Note: Prevent clickBubble so that removing a
                         contributor does not immediately un-remove her. -->
```

Comment:
We lost `clickBubble` here. Either we should bring it back, or remove this comment.